java/modules/core/src/main/java/org/apache/synapse/config/xml/FaultMediatorFactory.java

```diff
@@ -69,6 +71,7 @@
     private static final String SOAP12 = "soap12";
     private static final String POX = "pox";
 
+    @SuppressWarnings("startswith") //TRUE POSITIVE
```

Comment:
Why is this a true positive? Where even is the code that issues the error? This method is immense.